lib/t_web/controllers/mobile_auth_controller.ex

```diff
@@ -21,10 +21,13 @@ defmodule TWeb.MobileAuthController do
       token: Accounts.UserToken.encoded_token(token),
       user: render_user(user),
       # TODO proper screen_width
+      # TODO actually, should it be removed?
       profile:
         TWeb.ProfileView.render("show_with_location.json",
           profile: user.profile,
-          screen_width: 1000
+          screen_width: 1000,
+          # TODO update to 6.0.0
+          version: "5.2.3"
```

Comment:
that's why this should be indeed bumped to 6.0.0 (it's safe to decode 6.0.0 OwnProfile for 5.2.3 and below) because of an edge case where the only user's photo is in its private page (which won't be returned because of this)

Reply:
wouldn't ios crash on new stickers though?

Reply:
It's in 5.2.3:
https://github.com/getsince/test5/blob/add-seen-id-to-chat-version/Since/Story/Models/StoryLabel.swift#L106